#86 - Adding AI GATEWAY 9500-CM4 (#227)
geerlingguy/raspberry-pi-pcie-devices: _boards_cm/techbase-ai-gateway-9500-cm4.md

---
layout: board
title: "Techbase AI GATEWAY 9500-CM4"

> short_description: A CM4-based industrail gatway incorporating the Modberry 500-CM4 and Coral from Google.

Reviewer: ```suggestion
short_description: An industrial device incorporating the Modberry 500-CM4 and Coral from Google.
```

status: prototype
picture: "/images/board-ai-gateway-9500-cm4.jpg"
github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/86"
link: https://iiot-shop.com/product/ai-gateway/
videos: []
---

> The [AI GATEWAY 9500-CM4](https://modberry.techbase.eu/modberry/ai-gateway-series-with-compute-module-4-and-google-coral-tpu/) marries a CM4 with a Google Coral TPU, over PCIe (interestingly... I'm guessing they haven't been able to sell this through yet due to the PCIe driver bugs everyone's been running into—see [Issue #44](https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/44).

Reviewer: ```suggestion
The [AI GATEWAY 9500-CM4](https://modberry.techbase.eu/modberry/ai-gateway-series-with-compute-module-4-and-google-coral-tpu/) marries a CM4 with a Google Coral TPU, over PCIe (though this might not be working yet—see [Issue #44](https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/44).
```
